library/a-star/src/main/java/metadata/MetaData.java

```diff
@@ -0,0 +1,75 @@
+package metadata;
+
+import org.json.simple.JSONObject;
```

Comment:
I thought we discussed unifying the same json parser

Reply:
since this is a small feature, i believe its not too much of a priority right now and i havent had the time to compare both JSON apis

Reply:
I think this code needs unit testing before I can approve it, since this is no longer a tool/prototype but it is gonna be its own core module

Reply:
i thought we agreed this PR was about gradle integration and unit testing would be done subsequently